src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -16,6 +16,8 @@
 
 namespace OmniSharp.Script
 {
+    using Dotnet.Script.NuGetMetadataResolver;
```

Comment:
Please move using to be in the same block as the rest of the usings. Note that we don't nest them in namespaces in the OmniSharp source.